src/components/new-test/SuperSelect.tsx

```diff
@@ -0,0 +1,40 @@
+import {MenuItem, Select, Stack, Typography} from "@mui/material";
+import {SuperSelectType} from "../../Types/NewTestTypes";
+
+export const SuperSelect = ({title, value, items, handleChange}: SuperSelectType) => {
+
+    return <Stack direction={'column'} spacing={1}>
+        <Typography
+            sx={{
+                typography: 'subtitle1',
+                pl: '14px'
+            }}
+        >
+            {title}
+        </Typography>
+        <Select sx={{
+            width: 266,
+            height: 42,
+            bgcolor: 'primary.main',
+            color: 'primary.contrastText',
+            borderRadius: '15px',
+            typography: 'subtitle1',
+            '& .MuiOutlinedInput-notchedOutline': {
+                border: 0
+            },
+            '& .MuiSvgIcon-root': {
+                color: 'text.secondaryAlpha500'
+            },
+        }}
```

Comment:
This is the default select for our site, you need to add it to the theme (in this case select on home page must be secondary, that is, you need to rewrite the styles in the theme specifically for secondary)